### DISCUSSION TITLE
Detect if image is a document

### DISCUSSION THREAD

**Discussion by git-sources** [deepjavalibrary/djl#3439] [Q&A]
Hello,
is it possible to detect if a image is a document? How would I implement this? Which engine should I use?

Thanks
Marc

**Comment by frankfliu** [comment]
I think you need a image classification model. See: https://huggingface.co/tasks/image-classification

Most likely you have to train your own model. Once you trained your model you can use DJL to load the model. You can also take this example: https://github.com/deepjavalibrary/djl-demo/tree/master/footwear_classification
